src/Language/SystemVerilog/AST/Attr.hs

```diff
@@ -24,6 +25,14 @@ type AttrSpec = (Identifier, Expr)
 
 instance Show Attr where
     show (Attr specs) = printf "(* %s *)" $ commas $ map showSpec specs
+instance {-# OVERLAPPING #-} Show [Attr] where
+    show = foldr (\a b -> show a ++ " " ++ b) ""
+
+instance Semigroup Attr where
+    (<>) (Attr a1) (Attr a2) = Attr (a1 <> a2)
+
+instance Monoid Attr where
+    mempty = Attr []
```

Comment:
If we end up removing any usage of the `Monoid` and `Semigroup` instances, I'd prefer we remove their definitions as well.

Reply:
Thank you for pointing this out. I used it at some point but forgot to remove it.